statsmodels/tsa/vector_ar/tests/test_var.py

```diff
@@ -378,15 +382,15 @@ def test_acorr(self):
         acorrs = self.res.acorr(10)
 
     def test_forecast(self):
-        point = self.res.forecast(self.res.y[-5:], 5)
+        point = self.res.forecast(self.res.endog[-5:], 5)
 
     def test_forecast_interval(self):
-        y = self.res.y[:-self.p:]
+        y = self.res.endog[:-self.p:]
```

Comment:
Maybe I need more caffeine; what does this slice mean?

Reply:
The last colon is unnecessary